src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -658,6 +660,30 @@ void ApplyLiteralProperties()
 			}
 		}
 
+		/// <summary>
+		/// Override font properties at the end of the app.xaml ctor.
+		/// </summary>
+		/// <param name="writer"></param>
+		private void ApplyFontsOverride(IndentedStringBuilder writer)
+		{
+			var themes = new[] { "Default", "Light", "HighContrast" };
+
+			if(_generatorContext.GetMSBuildPropertyValue("UnoPlatformDefaultSymbolsFontFamily").Trim() is { Length: > 0 } fontOverride)
```

Comment:
Trim should be unnecessary. Compiler reads MSBuild properties from a generated editorconfig, and per editorconfig spec (which the compiler follows), the whitespaces are trimmed.

```suggestion
			if (_generatorContext.GetMSBuildPropertyValue("UnoPlatformDefaultSymbolsFontFamily") is { Length: > 0 } fontOverride)
```